src/main/java/study/calculator/Calculator.java

```diff
@@ -0,0 +1,111 @@
+package study.calculator;
+
+import java.util.*;
+
+import static study.calculator.Common.*;
+
+public class Calculator {
+
+    public static int plus(int num1, int num2) {
+        return num1 + num2;
+    }
+
+    public static int minus(int num1, int num2) {
+        return num1 - num2;
+    }
+
+    public static int multiply(int num1, int num2) {
+        return num1 * num2;
+    }
+
+    public static int divisionShare(int num1, int num2) {
+        return num1 / num2;
+    }
+
+    public static int divisionRemainder(int num1, int num2) {
+        return num1 % num2;
+    }
+
+    public static int getOperation(int num1, String operation, int num2) {
+
+        int result = 0;
+
+        switch (operation) {
+            case "+":
+                result = plus(num1, num2);
+                break;
+            case "-":
+                result = minus(num1, num2);
+                break;
+            case "*":
+                result = multiply(num1, num2);
+                break;
+            case "/":
+                result = divisionShare(num1, num2);
+                break;
+            default:
+                System.out.println("수식이 없습니다.");
+                break;
+        }
+
+        return result;
+    }
+
+    public static Map<String,List<String>> valueClassification(List<String> wordList) {
+        List<String> valueList = new ArrayList<>();
+        List<String> operationList = new ArrayList<>();
+
+        for(int i = 0; i< wordList.size(); i++){
+            
+            if (isEven(i)) {
+                addOperation(wordList.get(i), operationList);
+            }
+
+            if(!isEven(i)) {
+                addValue(wordList.get(i), valueList);
+            }
+        }
+
+        Map<String,List<String>> resultMap = new HashMap<>();
+        resultMap.put("values",valueList);
+        resultMap.put("operations",operationList);
+
+        return resultMap;
+    }
+
+    private static boolean isEven(int i) {
+        return i % 2 == 1;
+    }
+
+    private static void addValue(String value, List<String> valueList) {
+        checkValue(value);
+        valueList.add(value);
+    }
+
+    private static void addOperation(String value, List<String> operationList) {
+        checkOperation(value);
+        operationList.add(value);
+    }
+
+    public static int getResult(List<String> valueList, List<String> operationList) {
+
+        int result = 0;
+
+        for(int j = 0; j< valueList.size(); j++){
+
+            if(j>0){
```

Comment:
```suggestion
            if(j > 0){
```
이번 과정에서 들여쓰기는 1번 이하로 제한하고 있어요 들여쓰기를 줄이려면 어떻게 해야 할까요.

Reply:
enum 클래스를 활용해서 구조를 변경해보려고 준비중입니다.

https://techblog.woowahan.com/2527/
여기에서 약간의 힌트를 얻은 기분입니다.

enum과 Bifuntion을 이용해서 수정해보려고 하는데요. 어떻게 생각하시나요?